tests/nist/nist_test.sh

```diff
@@ -13,7 +13,19 @@ function test_nist {
 	test_dir="$1"
 	# make sure the output dir in builddir exists
 	mkdir -p "${builddir}/tests/nist/$test_dir/"
-	"${srcdir}/test_worker.py" --scanner "$OSCAP" --outputdir "${builddir}/tests/nist/$test_dir/" "${srcdir}/$test_dir/"
+	(
+		# workaround for OVAL 5.4 issue where var_ref regex pattern is too
+		# strict and doesn't allow numbers or underscores
+		if [ "x$test_dir" == "xR1100" ]; then
+			unset OSCAP_FULL_VALIDATION
+		fi
+		# workaround for missing idref in <platform> in an internally used
+		# XCCDF. this XCCDF is never exported and only used internally.
+		if [ "x$test_dir" == "xR1200" ]; then
+			unset OSCAP_FULL_VALIDATION
+		fi
+		"${srcdir}/test_worker.py" --scanner "$OSCAP" --outputdir "${builddir}/tests/nist/$test_dir/" "${srcdir}/$test_dir/"
+	)
 	ret_val=$?
```

Comment:
![image](https://cloud.githubusercontent.com/assets/753153/21782852/ef8e56bc-d682-11e6-9947-451516f76a7a.png)

This is gvim. Could it be that you have some custom broken syntax highlighting plugins? I just use vim inbuilt for bash.

```
$ gvim --version
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Dec 15 2016 16:47:14)
Included patches: 1-133
Modified by <bugzilla@redhat.com>
Compiled by <bugzilla@redhat.com>
```